lms/static/sass/bootstrap/_header.scss

```diff
@@ -0,0 +1,13 @@
+// Open edX: LMS header
+// ====================
+
+.header-global {
+  .logo-header {
+    margin: 0;
+    padding: 0;
+
+    img {
```

Comment:
@andy-armstrong why style the element instead of a class?

Reply:
Laziness. :-) There was no class on the image, and it didn't seem necessary to add one. I can change it.

Reply:
If you could that would be great.

Reply:
Done